src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/MoveToNamespaceDialog_OutOfProc.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+
+namespace Microsoft.VisualStudio.IntegrationTest.Utilities.OutOfProcess
+{
+    public class MoveToNamespaceDialog_OutOfProc : OutOfProcComponent
+    {
+        private const string MoveToNamespaceDialogId = "MoveToNamespaceDialog";
+
+        public MoveToNamespaceDialog_OutOfProc(VisualStudioInstance visualStudioInstance)
+            : base(visualStudioInstance)
+        {
+        }
+
+        /// <summary>
+        /// Verifies that the Move To Namespace dialog is currently open.
+        /// </summary>
+        public void VerifyOpen()
+        {
+            var dialog = DialogHelpers.FindDialogByAutomationId(GetMainWindowHWnd(), MoveToNamespaceDialogId, isOpen: true);
```

Comment:
@sharwell You introduced a new pattern for this right?

Reply:
Talked offline. Moving to new pattern